client-src/elements/chromedash-feature-detail.js

```diff
@@ -477,14 +497,27 @@ class ChromedashFeatureDetail extends LitElement {
     const name = `${processStage.name}${numberDifferentiation}`;
     const isActive = this.feature.active_stage_id === feStage.id;
 
+    // Show a button to add a trial extension stage for origin trial stages.
+    let addExtensionButton = nothing;
+    if (this.canEdit && 'extensions' in feStage) {
+      // Button text changes based on whether or not an extension stage already exists.
+      const extensionButtonText = (feStage.extensions && feStage.extensions.length > 0) ?
+        'Add another trial extension' : 'Add a trial extension';
+      addExtensionButton = html`
+      <sl-button size="small" style="float:right"
```

Comment:
In the screenshot it looked like this button needs some margin to separate it from the other button.

Reply:
Whoops, missed this before merging. I'll fix in another PR!